be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -48,6 +48,12 @@ OperatorPtr ConnectorScanOperatorFactory::do_create(int32_t dop, int32_t driver_
     return std::make_shared<ConnectorScanOperator>(this, _id, driver_sequence, dop, _scan_node);
 }
 
+const std::vector<ExprContext*>& ConnectorScanOperatorFactory::partition_exprs() const {
```

Comment:
Where you use `partition_exprs` this function?

Reply:
It is override from `SourceOperatorFactory`.